code/modules/research/nodes/combat.dm

```diff
@@ -203,16 +204,30 @@
 
 	unlocks_designs = list(/datum/design/research/item/weapon/mindflayer,  /datum/design/research/item/weapon/nuclear, /datum/design/research/item/weapon/lasercannon)
 
+/datum/technology/basic_armor
+	name = "Armor Solutions"
+	desc = "Standart issue armor of moebius paramedic teams."
```

Comment:
Should be "Standard"

Reply:
![image](https://user-images.githubusercontent.com/65828539/142850614-984d864c-d61e-48ad-90f3-7117959d3b19.png)